### DISCUSSION TITLE
Sequence of dimmer operations cause modal to be invisible

### DISCUSSION THREAD

**Discussion by indreksiitan** [fomantic/Fomantic-UI#1267] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Test case: https://jsfiddle.net/pb6nwvy3/3/

Expected behaviour:
* Progress indicator with dimmer is shown
* Progress indicator with dimmer is hidden
* Modal with message and background dimmer is shown

Actual behaviour:
* Message modal is created in DOM, but the enclosing dimmer is missing "active visible" classes, which renders it to be invisible.

**Comment by indreksiitan** [comment]
Made a much simpler example without our homegrown framework:
https://jsfiddle.net/xnga5rjs/

**Comment by lubber-de** [comment, marked as answer]
The modal displayer should be implemented to the dimmer hide callback.
Works then, See https://jsfiddle.net/yn9g5ju0/
